docs/rules/no-internal-modules.md

```diff
@@ -33,7 +38,8 @@ And the .eslintrc file:
   ...
   "rules": {
     "import/no-internal-modules": [ "error", {
-      "allow": [ "**/actions/*", "source-map-support/*" ]
+      "allow": [ "**/actions/*", "source-map-support/*" ],
+      "forbid": [ "**/actions/special/*" ],
```

Comment:
should it be possible to both allow and forbid specific things? When "allow" is specified, it takes precedence over a default "forbid" of "**/*". When "forbid" is specified, it takes precedence over a default "allow" of "**/*".

I'd expect the schema to only allow one or the other, never both.

Reply:
By 

> I'd expect the schema to only allow one or the other, never both.

You mean the user will either provide `allow` array or `forbid` array but never both ?

Reply:
Correct. If both are provided, we have to define a precedence between them - with `allow` of A and `forbid` of `B`, what's intended with `C`? Is it allowed, because it's not forbidden? Or is it forbidden, because it's not allowed?

Reply:
Yep. I understand now! I will rework the PR to do that 👍